### ISSUE TITLE
[BUG] `.read_csv` crash when read one line csv

### ISSUE THREAD

**Issue by xxsl** [fireducks-dev/fireducks#6]
To reproduce this bug:

First, save this content into a csv file (should be **just one line without `\n`**)
```csv
0,1
```

Then try to use `.read_csv` method of FireDucks to read this csv:
```python3
import fireducks.pandas as fpd
fdf = fpd.read_csv("bad.csv")

# Then any operation on this dataframe will crash, for example:
print(fdf) # <--- crash
```

The error when crash:
```python
2024-02-24 11:25:33.526492: 1630 fire[/lib/fire_helper.cc:21](http://172.17.0.2/lib/fire_helper.cc#line=20)] Encountered runtime error: Could not open CSV input source 'bad.csv': Invalid: CSV parse error: Empty CSV file or block: cannot infer number of columns
```

![2024-02-24_20-25](https://github.com/fireducks-dev/fireducks/assets/7067381/78141015-092f-4c3e-b2c7-8c0491cdb287)

Please note that Pandas can handle this without any problem:

![2024-02-24_20-26](https://github.com/fireducks-dev/fireducks/assets/7067381/e8a35126-9c72-486b-82ff-fae5ce3dc177)

Tested on:

- FireDucks version: 0.10.1
- OS: Ubuntu 22.04.3 LTS (GNU/Linux 6.1.0-17-amd64 x86_64)
- Python3 version: Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux

**Comment by xxsl** [comment]
Thanks for your explanations.
Since this is not intended to be fixed in short time, I shall close the issue too.

**Comment by qsourav** [comment]
@xxsl ,

Hope you are doing well. Thank you for reporting the issue. It is fixed in FireDucks 0.11.4.
